app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for entity in people:
+        person = Person(entity["name"], entity["age"])
+        if entity.get("wife"):
+            wife = Person.people.get(entity["wife"])
+            if wife:
+                person.wife = wife
+                wife.husband = person
+        elif entity.get("husband"):
+            husband = Person.people.get(entity["husband"])
+            if husband:
+                person.husband = husband
+                husband.wife = person
+        person_list.append(person)
```

Comment:
Split it into two loops:
1. Create all entities
2. Set husband/wife inside each people
For now it is unlogical. Because if we have: 
people = [{"name": "a", "age": 25, "wife": "b"}, {"name": "b", "age": 24}]
As a result we will have that a does not have any wife)